__docs__/wonder-blocks-card/card.argtypes.tsx

```diff
@@ -0,0 +1,56 @@
+import type {ArgTypes} from "@storybook/react";
+
+export default {
```

Comment:
Can we set up controls for the props that can only be specific values? (backgroundColor, padding, borderRadius, etc!) 

<img width="823" height="823" alt="Image" src="https://github.com/user-attachments/assets/2f03a091-7d4e-415b-9672-371d78af0965" />